.github/workflows/build-release.yml

```diff
@@ -3,6 +3,12 @@ name: "[builder] CI for releases"
 on:
   repository_dispatch:
     types: build
+  release:
+    types: [published]
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # UTC 17:00 -> CST (China) 1:00, see https://datetime360.com/cn/utc-cst-china-time/
+    - cron: '0 17 * * *'
```

Comment:
It doesn't seem like a good idea to release another nightly build.
https://github.com/HenryQW/Awesome-TTRSS/blob/e8006b2c373ba028ddd6b7df0a9b3c570ad880d5/.github/workflows/build-nightly-release.yml#L1-L5